bootstrap-salt.sh

```diff
@@ -4325,6 +4353,179 @@ install_freebsd_restart_daemons() {
 
 #######################################################################################################################
 #
+#   OpenBSD Install Functions
+#
+
+install_openbsd_deps() {
+  ping -c 2 -q mirrors.nycbug.org || return 1
+  echo "installpath = http://mirrors.nycbug.org/pub/OpenBSD/${OS_VERSION}/packages/${CPU_ARCH_L}/" >/etc/pkg.conf || return 1
+  pkg_add -I -v lsof || return 1  
+  pkg_add -I -v py-pip || return 1  
+  ln -sf $(ls -d /usr/local/bin/pip2.*) /usr/local/bin/pip  || return 1 
+  ln -sf $(ls -d /usr/local/bin/pydoc2*) /usr/local/bin/pydoc || return 1 
+  ln -sf $(ls -d /usr/local/bin/python2.[0-9]) /usr/local/bin/python || return 1
+  ln -sf $(ls -d /usr/local/bin/python2.[0-9]*to3) /usr/local/bin/2to3 || return 1
+  ln -sf $(ls -d /usr/local/bin/python2.[0-9]*-config) /usr/local/bin/python-config || return 1
+  pkg_add -I -v swig || return 1
+  pkg_add -I -v py-zmq || return 1
+  pkg_add -I -v py-requests || return 1
+  pkg_add -I -v py-M2Crypto || return 1
+  pkg_add -I -v py-raet || return 1
+  pkg_add -I -v py-libnacl || return 1
+  if [ "$_UPGRADE_SYS" -eq $BS_TRUE ]; then
+      /usr/local/bin/pip install --upgrade pip || return 1
+  fi
+  #
+  # PIP based installs need to copy configuration files "by hand".
+  # Let's trigger config_salt()
+  #
+  if [ "$_TEMP_CONFIG_DIR" = "null" ]; then
+      # Let's set the configuration directory to /tmp
+      _TEMP_CONFIG_DIR="/tmp"
+      CONFIG_SALT_FUNC="config_salt"
+      for fname in minion master; do
+          # Skip if not meant to be installed
+          [ $fname = "minion" ] && [ "$_INSTALL_MINION" -eq $BS_FALSE ] && continue
+          [ $fname = "master" ] && [ "$_INSTALL_MASTER" -eq $BS_FALSE ] && continue
+          # [ $fname = "api" ] && ([ "$_INSTALL_MASTER" -eq $BS_FALSE ] || [ "$(which salt-${fname} 2>/dev/null)" = "" ]) && continue
+          [ $fname = "syndic" ] && [ "$_INSTALL_SYNDIC" -eq $BS_FALSE ] && continue
+
+          # Let's download, since they were not provided, the default configuration files
+          if [ ! -f "$_SALT_ETC_DIR/$fname" ] && [ ! -f "$_TEMP_CONFIG_DIR/$fname" ]; then
+              ftp -o "$_TEMP_CONFIG_DIR/$fname" \
+                  "https://raw.githubusercontent.com/saltstack/salt/develop/conf/$fname" || return 1
```

Comment:
@serge-p, this should probably be using `$STABLE_REV`: `https://raw.githubusercontent.com/saltstack/salt/${STABLE_REV}/conf/master`.